text/3392-leadership-council.md

```diff
@@ -6,11 +6,11 @@
 
 # 摘要
 
-This RFC establishes a Leadership Council as the successor of the core team[^core] and the new governance structure through which Rust Project members collectively confer the authority[^authority] to ensure successful operation of the Project. The Leadership Council delegates much of this authority to teams (which includes subteams, working groups, etc.[^teams]) who autonomously make decisions concerning their purviews. However, the Council retains some decision-making authority, outlined and delimited by this RFC.
+這份請求意見書（RFC）確立了一個領導理事會，作為核心團隊的繼任者[^core]和 Rust 專案的新治理結構，透過這個結構，Rust 專案成員共同授權[^authority]以確保專案的成功運作。領導理事會將大部分的權限委派給各個團隊（包括子團隊、工作小組等[^teams]），讓他們自主決策有關其範圍的事項。但理事會仍然保留一些決策權限，由這份請求意見書來明確和界定。
```

Comment:
I am not aware of our translating "RFC" elsewhere. Should we not translate it at all?